src/sbvr-api/sbvr-utils.coffee

```diff
@@ -540,14 +541,34 @@ exports.runRule = do ->
 		.nodeify(callback)
 
 exports.PinejsClient =
-	class PinejsClient extends PinejsClientCore(_, Promise)
-		_request: ({ method, url, body, tx, req, custom }) ->
+	class PinejsClient extends PinejsClientCore(_, Promise, mimemessage)
+		compileBatch: (data) -> _.map(data, batchify)
+		_request: ({ method, url, body, multipart, tx, req, custom }) ->
+			if _.isArray(multipart?.data)
+				body = multipart.data
 			return runURI(method, url, body, tx, req, custom)
 
+batchify = (req) ->
```

Comment:
Could you change `req` to `data` or `body` or similar, I expect `req` to refer specifically to en expressjs req object and based on `compileBatch: (data) -> _.map(data, batchify)` it's not